src/main/java/genji/Genji.java

```diff
@@ -0,0 +1,55 @@
+package genji;
+
+import genji.command.Command;
+import genji.task.TaskList;
+
+/**
+ * The main class of the chatbot.
+ * Includes constructor, run method, and main method
+ */
+public class Genji {
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+    private static String FILE_PATH = "./data/Genji.txt";
+
+    /**
+     * Constructor of Genji
+     * @param filePath place store and load the list
+     */
+    public Genji(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        this.taskList = storage.loadList();
+    }
+
+    /**
```

Comment:
Adding on to the JavaDocs section, maybe you can consider adding the parameters and the result to the function as well. It might make the description of the code more comprehensive and easily understandable by other developers. For example, just by doing /** and entering, the template for the method javadocs is automatically shown in code editors like IntelliJ.